src/components/editions/headline/index.tsx

```diff
@@ -31,11 +31,28 @@ const tablet = tabletContentWidth + 12;
 
 // ----- Template Format Specific Styles ----- //
 
-const analysisStyles = (kickerColor: string): SerializedStyles => css`
-	text-decoration: underline;
-	text-decoration-thickness: from-font;
-	text-decoration-color: ${kickerColor};
-	padding-bottom: 0;
+const analysisStyles = css`
+	background-image: repeating-linear-gradient(
+		to bottom,
+		transparent,
+		transparent 46px,
+		${neutral[86]}
+	);
+	line-height: 48px;
```

Comment:
I'd suggest `rem`s for this to avoid scaling issues (assuming Editions uses the same scaling mechanism as AR).

Reply:
Updated.